### ISSUE TITLE
[Tokenizer]: Clear All button a11y issue -  cannot be accessed by keyboard

### ISSUE THREAD

**Issue by Shike-m** [UI5/webcomponents#13940]
### Describe the bug

When both multiLine and showClearAll are set to true. the "Clear All" button is presented. It cannot be navigated to by keyboard. 

### Isolated Example

https://ui5.github.io/webcomponents-react/v2/?path=/docs/inputs-tokenizer--docs

### Reproduction steps

1. In the official example I provided, you can just enable the two properties.
2. The "Clear All" is presented, you can test it here. Or any other examples for this point.
3. Screen readers do not work too.
4. A final customer reported the issue. 

### Expected Behaviour

Work with keyboard navigation and screen readers.

### UI5 Web Components for React Version

2.25.0

### UI5 Web Components Version

2.25.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by Vonahz** [comment]
Hi @Shike-m,

Thanks for reporting this. I've changed the type to feature because the tokenizer "Clear All" button is intentionally not keyboard-focusable per design specification. We'll treat this as a feature request and look into it.

Best regards,
Milen

**Comment by OlgaFlaccus** [comment]
Hi all, 

internal BLI created. 

Best, 
Olga

**Comment by OlgaFlaccus** [comment]
Hi @Shike-m ,

this is expected behavior since the keyboard users can achieve the same functionality by pressing ctrl/cmd + A and del-key. so it works as per design. 

best, 
Olga
